### DISCUSSION TITLE
Version 1.24.2

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#360] [Announcements]
- Bugfixes and optimizations

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.24.2'>Version 1.24.2</a>.</em>

**Comment by sairamkumarm** [comment]
Hey, great app! It's my new daily driver.
But how do I update it? Do I delete it and do a fresh install? If so, what happens to my tags and settings? I am still kinda new to this. Sorry if my question is too basic.

**Reply by MM2-0** [reply]
Just install the new version, no need to uninstall thr previous versiom first. But if you do want to do a clean install, you can use the launcher's backup and restore functionality to keep your data.

**Reply by sairamkumarm** [reply]
Oh, got it. Thank you!
